NorthstarDedicatedTest/cl_rcon.cpp

```diff
@@ -0,0 +1,365 @@
+//===========================================================================//
+//
+// Purpose: Implementation of the rcon client.
+//
+//===========================================================================//
+
+#include "pch.h"
+#include "convar.h"
+#include "concommand.h"
+#include "cvar.h"
+#include "rcon_shared.h"
+#include "sv_rcon.pb.h"
+#include "cl_rcon.pb.h"
+#include "cl_rcon.h"
+#include "net.h"
```

Comment:
For some reason, this include causes the CI build to fail.

Removing the include (and the one function call that needs it) causes the CI to build just fine (excluding formatting errors).

See also 1b6e0aa3d8f2774a68ef361141a0c4b7e411f3b2

1b6e0aa3d8f2774a68ef361141a0c4b7e411f3b2

Reply:
Fixed as of 4573d5e3141915f7066f182b47dac6a1d7d7bdcb.
This was caused by commit 938d89713eb055340b5e8866de3000b61e4a0f13 from the Squirrel PR as found by Gecko.